kibana-proxy/Dockerfile

```diff
@@ -1,17 +1,11 @@
 FROM alpine:latest
 
 RUN apk update
-RUN apk add --no-cache nginx wget openrc
-
-# RUN yum update -y && amazon-linux-extras install -y nginx && yum install -y wget
+RUN apk add --no-cache wget
 
 RUN wget https://github.com/abutaha/aws-es-proxy/releases/download/v0.9/aws-es-proxy-0.9-linux-386
 
-RUN mv /etc/nginx/nginx.conf /etc/nginx/nginx.conf.save
-
-ADD nginx.conf /etc/nginx/
-
-RUN rc-update add nginx default
+RUN chmod u+x /aws-es-proxy-0.9-linux-386
 
 EXPOSE 80
-ENTRYPOINT ["/aws-es-proxy-0.9-linux-386 -endpoint ${ELASTICSEARCH_ENDPOINT} -no-sign-reqs -listen 0.0.0.0:9200"]
+CMD /aws-es-proxy-0.9-linux-386 -verbose -endpoint "${ELASTICSEARCH_ENDPOINT}" -no-sign-reqs -listen 0.0.0.0:80
```

Comment:
Do we want to leave `--verbose` in by default? (or should we do something like `${AWS_ES_PROXY_ARGS:-}` to allow adding it through the environment?